jobrunner/bashrc

```diff
@@ -1,7 +1,9 @@
 # load config into this shell so we can just run stuff
 set -a
 # shellcheck disable=SC1090
-. /srv/jobrunner/environ/*.env
+for f in /srv/jobrunner/environ/*.env; do
+    . $f
```

Comment:
should this be `. "$f"` or it doesn't matter? (nb. how come this doesn't trigger https://github.com/koalaman/shellcheck/wiki/sc2086 ?)